utt/report/details/view.py

```diff
@@ -50,6 +53,26 @@ def render(self, output: Output) -> None:
 
         print(file=output)
 
+    def csv(self, output: Output) -> None:
+        if not self._model.activities:
+            print(" -- No activities for this time range --", file=output)
+            return
+
+        fieldnames = ["task", "project", "date", "duration", "type", "comment"]
```

Comment:
In https://github.com/larose/utt/blob/e4143d6d120085e3e8b653053dced3837e655d48/utt/report/per_day/csv_view.py#L17 the usage of the plural form is applied to hours, projects and tasks. We should be consistent. Either we change to plural here or change to singular in `per_day/csv_view.py`. 

Also, let's arrange the columns in a more logical order, moving from general to specific. For instance, let's reorder the first three columns to be "date" "project" and "task" in that sequence.